### PR TITLE
[msal-node] adding READMEs to msal-node b2c samples

### PR DESCRIPTION
**Pull request by derisen**
This PR adds a README.md file to each of the MSAL Node B2C samples.

### CODE REVIEW

File: samples/msal-node-samples/standalone-samples/b2c-auth-code-pkce/README.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+        }
+    }
+};
+```
+
+MSAL enables PKCE in the Authorization Code Grant Flow by including the `codeChallenge` and `codeChallengeMethod` parameters in the request passed into `getAuthCodeUrl()` API, as well as the `codeVerifier` parameter in the second leg (`acquireTokenByCode()` API).
+
+Generating the `codeVerifier` and the `codeChallenge` is the client application's responsibility. For this sample, you can either implement your own PKCE code generation logic or use an existing tool to manually generate a **Code Verifier** and **Code Challenge**, plugging them into the `pkceCodes` object below.
+
+For details on implementing your own PKCE code generation logic, consult the PKCE specification `https://tools.ietf.org/html/rfc7636#section-4`
```
Comment (sameerag) [inline samples/msal-node-samples/standalone-samples/b2c-auth-code-pkce/README.md:74]: @samuelkubai updated that our PKCE helpers are indeed functional in node. Can @hectormmg / @derisen please confirm the same for `msal-node` in AAD and B2C samples? We can update the docs once we are confident.

**Comment by jasonnutter** [comment]
@derisen @sameerag @samuelkubai @hectormmg This PR has been open for a couple weeks, can we please work to get it merged?

File: samples/msal-node-samples/standalone-samples/b2c-auth-code-pkce/README.md (hunk 1 above)
Comment (samuelkubai) [reply samples/msal-node-samples/standalone-samples/b2c-auth-code-pkce/README.md:74]: We found out that there's a bug that's causing some inconsistency with the result of the PKCE helpers that is to be fixed with this [pull request ](https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/2687)

**Comment by hectormmg** [comment]
> @derisen @sameerag @samuelkubai @hectormmg This PR has been open for a couple weeks, can we please work to get it merged?

@jasonnutter sorry for the delay here, I'll update this sample as soon as we get #2687 merged!

**Review by pkanher617 — APPROVED** [review]

**Comment by sameerag** [comment]
Now that the dependency PR is merged, can we please update the docs and merge this PR? @samuelkubai Can you please address this?

**Comment by samuelkubai** [comment]
@sameerag I can, I will pick it up.

File: samples/msal-node-samples/standalone-samples/b2c-auth-code-pkce/README.md (hunk 1 above)
Comment (samuelkubai) [reply samples/msal-node-samples/standalone-samples/b2c-auth-code-pkce/README.md:74]: @sameerag I have added an instruction to use our `CryptoProvider` to generate the PKCE codes.

Comment (sameerag) [reply samples/msal-node-samples/standalone-samples/b2c-auth-code-pkce/README.md:84]: Thanks @samuelkubai. Looks good.

**Comment by sameerag** [comment]
@samuelkubai Can you please address what is pending here and merge it?

File: samples/msal-node-samples/standalone-samples/b2c-auth-code/README.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+```
+
+In `index.js`, we setup the configuration object expected by MSAL Node `confidentialClientApplication` class constructor:
+
+```javascript
+const confidentialClientConfig = {
+    auth: {
+        clientId: "e6e1bea3-d98f-4850-ba28-e80ed613cc72",
+        authority: policies.authorities.signUpSignIn.authority, //signUpSignIn policy is our default authority
+        clientSecret: "wr7Q9R~Gg1Qb9l.3s4Dg8jp7Z6.9M~42K0",
```
Comment (hectormmg) [inline samples/msal-node-samples/standalone-samples/b2c-auth-code/README.md:55]:
```suggestion
        clientSecret: "CLIENT_SECRET",
```

**Review by hectormmg — APPROVED** [review]

**Pull request merged**